dotcom-rendering/src/components/AnalyticsIframe.amp.tsx

```diff
@@ -17,7 +17,7 @@ export const AnalyticsIframe = ({ url }: Props) => {
 				return (
 					<amp-iframe
 						class={prebidIframeStyle}
-						data-block-on-consent="_till_accepted"
+						data-block-on-consent="_till_responded" // TODO: TO CONFIRM
```

Comment:
Is this TODO still open?

Reply:
yes, I'm currently documenting the behaviour for all the changes. https://docs.google.com/document/d/1qo2CY8cgIe5RAiqH1iP6daa-ixuBcD70s2BR8nHmIJU/edit?usp=sharing